tensortrade/environments/trading_environment.py

```diff
@@ -224,14 +264,16 @@ def reset(self) -> np.array:
             The episode's initial observation.
         """
         self.episode_id = uuid.uuid4()
-
         self.clock.reset()
         self.feed.reset()
         self.action_scheme.reset()
         self.reward_scheme.reset()
         self.portfolio.reset()
         self.history.reset()
         self._broker.reset()
+        for r in self._renderer:
+            if r.can_reset:
```

Comment:
And we can remove this `can_reset` check

Reply:
I thought of implementing `can_reset()` and `can_save()` as just `pass` in the base class method - which is a smoother approach - but thought it could be misleading if a developer uses the classes directly expecting results from these two methods but gets nothing so it would be safer to receive an explicit error. I'm still leaning towards the `pass` approach though. So we have either an explicit error or silent "do nothing". What do you think?

Reply:
Ok, I went ahead and changed it in f2cceb7 to expedite the merge. We can revert later if we settle on the other opinion.